server/issue.go

```diff
@@ -1035,6 +993,42 @@ func createGhostTaskList(database *api.Database, vcsPushEvent *vcs.PushEvent, de
 	return taskCreateList, taskIndexDAGList, nil
 }
 
+func checkCharacterSetCollationOwner(dbType db.Type, characterSet, collation, owner string) error {
```

Comment:
Prefer `validateInstanceForDatabaseCreate()`, and pass in `instance` and `api.CreateDatabaseContext` as is, which introduce the lease code changes to the original logic.
The current rewrite is a bit hard to validate whether it's the same as the original implementation.

Reply:
Whether to support character set, collation and owner is only related to `dbType`. I don't think it's necessary to introduce an entire `instance` or `api.CreateDatabaseContext`.
cc @d-bytebase

Reply:
Either way works.